packages/dev/gui/src/2D/controls/rectangle.ts

```diff
@@ -29,6 +31,18 @@ export class Rectangle extends Container {
     public get cornerRadius(): number {
         return this._cornerRadius;
     }
+    public get cornerRadiusX(): number {

```

Comment:
We want to add `@serialize()` to these properties (like in `cornerRadius`) to be able to save and load them.